packages/trip-form/i18n/vi.yml

```diff
@@ -27,10 +30,10 @@ otpUi:
     walkReluctanceHigh: Thêm phương tiện công cộng
     walkReluctanceLow: Đi bộ nhiều hơn
     walkSpeed: Tốc độ đi bộ
-    watts: Sức mạnh xe tay ga điện
-    watts125kidsHoverboard: Ván trượt của trẻ em (10 km/h)
-    watts1500powerfulEscooter: Xe tay ga điện mạnh mẽ (40 km/h)
-    watts250entryLevelEscooter: Xe tay ga cấp nhập cảnh (18 km/h)
-    watts500robustEscooter: Xe tay ga điện mạnh mẽ (30 km/h)
-    wheelchair: Thích các tuyến xe lăn có thể truy cập
+    watts: Công suất xe tay ga điện
+    watts125kidsHoverboard: Ván trượt của trẻ em (6 mph)
+    watts1500powerfulEscooter: Xe tay ga điện công suất mạnh (24 mph)
```

Comment:
Is changing to mph intentional?

Reply:
No... I have reverted them back in Weblate, they'll trickle down here shortly.